### PR TITLE
feat:[Fermi] add Snippets plugin for Flow Launcher

### PR DESCRIPTION
**Pull request by Fermiz**
This Snippets plugin Flow launcher let you save key/value snippets and copy to clipboard copy it. Check on https://github.com/Fermiz/Flow.Launcher.Snippets

### CODE REVIEW

**Comment by Fermiz** [comment]
Hi @KrystianLesniak, can anyone review this PR? it's has been a long time since I sent the PR, I am new to this plugin publish process, any steps or procedures did I missed?

**Comment by KrystianLesniak** [comment]
Hello @Fermiz!
I'm sorry, but I'm not a member of the Flow Launcher team, so I'm not able to review and accept your pull request 😆. You need to contact someone else in this regard: https://github.com/orgs/Flow-Launcher/people. 

However, I have reviewed your entry and it seems that you have 2 redundant fields: `"DateAdded"` and `"LatestReleaseDate"`, you can remove them. They are automatically added by the CI workflow job after the merge process.

These are only required entries:
```json
{
  "ID": "This is a unique GUID for your plugin and needs to be the same as the ID in your plugin.json",  
  "Name": "Plugin name",
  "Description": "Short description of your plugin",
  "Author": "Author of the plugin",
  "Version": "Version of your plugin and needs to be the same as the Version in your plugin.json",
  "Language": "The programming language the plugin is written in",
  "Website": "Your plugin's website",
  "UrlDownload":"The url to download the plugin",
  "UrlSourceCode": "Url to the source code of the plugin",
  "IcoPath": "Url to the icon of the plugin"
}
```

Otherwise everything looks fine regarding plugin.json.

Take care and good luck with your PR! 😉

**Comment by jjw24** [comment]
Hi @Fermiz, apologies for the delay in reviewing. I will have a look tomorrow. In the meantime please update the json file per KrystianLesniak's comment.

@KrystianLesniak thanks for helping out :)

File: plugins.json [hunk 1]
```diff
@@ -1663,5 +1663,19 @@
...
+        "Description":"Simple plugin to save key/value snippets and copy to clipboard",
+        "Author":"Fermi Shuangqi Li",
+        "Version":"1.0.0",
+        "Language":"python",
+        "Website":"https://github.com/Fermiz/Flow.Launcher.Snippets",
+        "UrlDownload": "https://github.com/Fermiz/Flow.Launcher.Snippets/releases/download/v1.0.1/Flow.Launcher.Plugin.Snippets.zip",
+        "UrlSourceCode": "https://github.com/Fermiz/Flow.Launcher.Snippets/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/Fermiz/Flow.Launcher.Snippets/assets/snippets.png",
+        "DateAdded": "2023-08-23T15:30:00Z",
+        "LatestReleaseDate": "2023-08-23T15:00:00Z"
```
Comment (jjw24) [inline plugins.json:1679]:
```suggestion
        "Version":"1.0.1",
        "Language":"python",
        "Website":"https://github.com/Fermiz/Flow.Launcher.Snippets",
        "UrlDownload": "https://github.com/Fermiz/Flow.Launcher.Snippets/releases/download/v1.0.1/Flow.Launcher.Plugin.Snippets.zip",
        "UrlSourceCode": "https://github.com/Fermiz/Flow.Launcher.Snippets/tree/main",
        "IcoPath": "https://cdn.jsdelivr.net/gh/Fermiz/Flow.Launcher.Snippets/assets/snippets.png"
```

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
Just FYI, I noticed the show message call after saving/copying is not working.

**Comment by Fermiz** [comment]
Hi @jjw24 thank you for the comments, I would publish the fix later next week to move the data storage into  UserData's plugin settings folder

**Pull request merged**